TodoList/TodoList.Api.Tests/Utils/ExtensionItemCompare.cs

```diff
@@ -0,0 +1,12 @@
+﻿using NUnit.Framework.Constraints;
+
+namespace TodoList.Api.Tests.Utils
+{
+    public static class ExtensionItemCompare
+    {
+        public static EqualConstraint UsingItemCompare(this EqualConstraint equalConstraint)
```

Comment:
You don't need to create a new instance of the comparer each time the method is called. You know your implementation and you know it is a class assembled from pure methods - for same input it will always return same output…

Reply:
Keep in mind that test assemblies are loaded twice in `NUnit`- once to discover all tests and once to run (given) tests. During the first load, comparer does not need to be instantiate since it will never be used. Also, if the tests assembly grows and only some tests are executed, the comparer might not be needed even during the second load (~ test run). While it is absolutely not crutial in this case (is the comparer fairly small), let's leverage `Lazy` class and try to mitigate the unncessary initialization (gaining thread-safety as bonus) 😃